io/disk/htx_block_devices.py

```diff
@@ -118,8 +118,10 @@ def test(self):
         self.log.info("Running the HTX on %s", self.block_device)
         cmd = "htxcmdline -run %s -mdt %s" % (self.block_device, self.mdt_file)
         process.system(cmd, ignore_status=True)
-        for _ in range(0, self.time_limit, 60):
-            self.run_smt()
+        for time_loop in range(0, self.time_limit, 60):
+            # Running SMT changes every hour
+            if time_loop % 3600 == 0:
```

Comment:
I had a discussion with developers, and it is not advisable to change SMT values so frequently when a workload ia running.
So, a sufficient time interval (1 hour in this case) is necessary.